config/constants.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Constants
+  SEED_ADMIN_EMAIL = ENV.fetch("SEED_ADMIN_EMAIL", nil)
+  SEED_ADMIN_INITIAL_PASSWORD = ENV.fetch("SEED_ADMIN_INITIAL_PASSWORD", nil)
+  MAIL_FROM = ENV.fetch("MAIL_FROM", nil)
```

Comment:
A lot of these should be mandatory, like so

```suggestion
  MAIL_FROM = ENV.fetch("MAIL_FROM")
```